src/Types/types.tsx

```diff
@@ -0,0 +1,34 @@
+export type SideBarOption ={
+    id: number,
+    name: string,
+    icon: JSX.Element
+}
+
+export type ChatListDataType = {
+    id: number,
+    name: string,
+    lastMessage: string,
+    lastModified: string
+}
+
+export type ChatMessageType = {
+    id: number,
+    sender: number,
+    message: string,
+    timing: string
+}
+
+export type UserMessagesResType = {
+    id: string,
+    messages: ChatMessageType[]
+}
+
+export type Action = {
+    type: string,
+    newMessage: ChatMessageType
+}
+
+export type AvatarInfoType ={
+    background: string,
+    text: string 
+}
```

Comment:
We should not keep all the types in the top level.

**For example:** SideBarOption type should be insider SideBar component directory.

We should not append **Type** in type name.
In cases when type name is clashing with component name we should alias type while importing it.